pyproject.toml

```diff
@@ -46,6 +46,7 @@ ruamel-yaml-clib = [
     { version = "0.2.8", markers = "sys_platform == 'darwin' and platform_machine == 'x86_64'" }
 ]
 myst-nb = "^1.3.0"
+platformdirs = "^4.3.8"
```

Comment:
The `platformdirs` is a new one for me. Looks like a cool package and reduces some of the boilerplate for what you're doing here which is great.